src/common/outputService.ts

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable no-console */
+
+import { Connection } from '@salesforce/core';
+import { AsyncOperationResult, ChangeBundleInstall, WorkItemPromote } from './types';
+import { DeploySummaryQueryResult, selectDeployAORSummaryData } from './selectors/deployProgressSummarySelector';
+import { selectOrgUrl } from './selectors/endpointSelector';
+import { EnvQueryResult, selectPipelineStageByEnvironment } from './selectors/environmentSelector';
+import { WorkItemsQueryResult, selectWorkItemsByChangeBundles } from './selectors/changeBundleSelector';
+
+/**
+ * This type is used for storing all the info needed for the deployment summary
+ */
+type DeploySummary = {
+  stageName: string;
+  // It is a map if it is a soup or versioned promote
+  // It is an array of strings if it is an ad hoc promote
+  workItems: Map<string, string[]> | string[];
+  branchName: string;
+  orgUrl: string;
+};
+
+/**
+ * Service class to print the output
+ *
+ * @author JuanStenghele-sf
+ */
+export class OutputService {
+  protected con: Connection;
+
+  public constructor(con: Connection) {
+    this.con = con;
+  }
+
+  public static printAsyncRun(aorId: string): void {
+    console.log(`Job ID: ${aorId}/n
+      Deploy has been queued./n
```

Comment:
This is very specific for the deploy, I think that maybe we will have different messages for other operations with AORs